tests/unit/alloc_test.cc

```diff
@@ -19,6 +19,7 @@
  * Copyright (C) 2015 Cloudius Systems, Ltd.
  */
 
+#include <boost/test/tools/old/interface.hpp>
```

Comment:
This is the first time I see this include. Is it the right one?

Reply:
I mean it is probably the one that declares some method/helper I used but it is probably not the "right way" to include that but an internal detail: likely a result of my editor doing ICWYU and guessing wrong. I will remove it and sorry for the noise.